.gitignore

```diff
@@ -3,12 +3,5 @@ vendor
 <<<<<<< HEAD
 <<<<<<< HEAD
```

Comment:
Check this file again plz

Reply:
https://github.com/xiaoxiaoking/medusa/commit/de1a3fdfdb9877d2c8de4b6c7e14628db240b8ec